### DISCUSSION TITLE
Invalid or expired user code?

### DISCUSSION THREAD

**Discussion by rgl** [netbirdio/netbird#660] [Q&A / Support]
I'm trying out Netbird 0.12.0 on a Windows machine (buried in a network with an unknown topology), and for some reason it failed to Connect with the "Invalid or expired user code":

![image](https://user-images.githubusercontent.com/43356/213790713-bbdeaedc-5d20-4e15-9e7c-eb9e17271a99.png)

That page is automatically started from the Netbird UI application, so I'm kinda lost on how to proceed. What can I do to troubleshoot this?

**Comment by rgl** [comment]
Oh, this is awkward, I've just tried it again (after trying several time before submitting this plea for help), and its now working! Not sure what was going on.

**Comment by krombel** [comment]
I have the same issue right now. I think it came from a failed login attempt I had before.
I think that is timeout based (although I dont know how long it might take...)

My version: 0.22.7
